Jenkinsfile

```diff
@@ -15,13 +15,15 @@ def transformIntoStep(architecture, dockerImage, buildScript) {
     return {
         timeout(120) {
             def wspwd = pwd()
+            def ccache_dir_host = "$HOME/.ccache-$architecture"
             dir("build-$architecture") {
                 sh 'touch .git-keep'
+                sh "mkdir -p '$ccache_dir_host'"
                 withDockerRegistry( registry: [credentialsId: dockerCredentials, url: 'https://'+dockerRegistry ] ) {
                     withDockerContainer(
                         image: dockerRegistry+dockerImage,
                         args: " \
-                            -v $HOME/.ccache:/.ccache \
```

Comment:
it looks like apple does not use cache at all:

```
cache directory                     .ccache-apple.x86_64/
primary config                      .ccache-apple.x86_64//ccache.conf
secondary config (readonly)         /etc/ccache.conf
cache hit (direct)                     0
cache hit (preprocessed)               0
cache miss                             0
cache hit rate                      0.00 %
cleanups performed                     0
files in cache                         0
cache size                           0.0 kB
max cache size                       5.0 GB
```

I'll probably have too look if ccache-lipo is used properly in the runner image